### PR TITLE
Updated articles

### PR DESCRIPTION
**Pull request by umairsy**

### CODE REVIEW

File: erpnext/docs/user/manual/en/selling/articles/applying-discount.md [hunk 1]
```diff
@@ -1,31 +1,25 @@
-<h1>Applying Discount</h1>
+#Applying Discount
 
-There are two ways Discount can be applied on an items in the sales transactions.
+There are several ways Discount can be applied on an items in the sales transactions.
 
 #### 1. Discount on "Price List Rate" of an item
 
-In the Item table of transaction, after Price List Rate field, you will find Discount (%) field. Discount Rate applied in this field will be applicable on the Price List Rate of an item.
+In the Item table, after Price List Rate field, you will find Discount (%) field. Discount Rate applied will be applicable on the Price List Rate of an item.
```
Comment (anandpdoshi) [inline erpnext/docs/user/manual/en/selling/articles/applying-discount.md]: Simplified:
*You can find the Discount (%) field in the Item table. Discount is applied on the Price List Rate to get the selling Rate of the Item.*

File: erpnext/docs/user/manual/en/selling/articles/applying-discount.md [hunk 2]
```diff
@@ -1,31 +1,25 @@
-<h1>Applying Discount</h1>
+#Applying Discount
 
-There are two ways Discount can be applied on an items in the sales transactions.
+There are several ways Discount can be applied on an items in the sales transactions.
```
Comment (anandpdoshi) [inline erpnext/docs/user/manual/en/selling/articles/applying-discount.md]: Typo *on an items* should **on an item**

File: erpnext/docs/user/manual/en/selling/articles/applying-discount.md [hunk 3]
```diff
@@ -1,31 +1,25 @@
...
 #### 1. Discount on "Price List Rate" of an item
 
-In the Item table of transaction, after Price List Rate field, you will find Discount (%) field. Discount Rate applied in this field will be applicable on the Price List Rate of an item.
+In the Item table, after Price List Rate field, you will find Discount (%) field. Discount Rate applied will be applicable on the Price List Rate of an item.
 
-Before applying Discount (%). 
+<img alt="Discount Percentage" class="screenshot" src="{{docs_base_url}}/assets/image/articles/discount-1.png">
 
-![Before discount]({{docs_base_url}}/assets/img/articles/Selection_00616c670.png)
+The feature of % Discount is available in all the sales and purchase transactions.
```
Comment (anandpdoshi) [inline erpnext/docs/user/manual/en/selling/articles/applying-discount.md]: *... all ~~the~~ sales and ...*

File: erpnext/docs/user/manual/en/selling/articles/applying-discount.md [hunk 4]
```diff
@@ -1,31 +1,25 @@
...
 
-After applying Discount (%) under Discount on Price List Rate (%) field.
+You can also %Discount applied based on **Pricing Rule** as well. [Click here to learn how Pricing Rule functions.]({{docs_base_url}}/user/manual/en/accounts/pricing-rule.html)
```
Comment (anandpdoshi) [inline erpnext/docs/user/manual/en/selling/articles/applying-discount.md]: Better version:
*You can use Pricing Rule to automatically apply Discount (%) ... how to use Pricing Rule.*

File: erpnext/docs/user/manual/en/selling/articles/applying-discount.md [hunk 5]
```diff
@@ -1,31 +1,25 @@
...
 
-![After discount]({{docs_base_url}}/assets/img/articles/Selection_007f81dc2.png)
-     
-You can apply percent discount in all sales and purchase transactions.
+#### 2. Discount on Net Total and Grand Total
 
-#### 2. Discount on Grand Total
+From the "Additional Discount" section, you can apply flat discount, as amount or as percentage.
```
Comment (anandpdoshi) [inline erpnext/docs/user/manual/en/selling/articles/applying-discount.md]: *In the "Additional Discount" section*

Comment (umairsy) [inline erpnext/docs/user/manual/en/selling/articles/applying-discount.md]: thanks. fixed till now. after you complete review, will resend pull request.

File: erpnext/docs/user/manual/en/selling/articles/applying-discount.md [hunk 6]
```diff
@@ -1,31 +1,25 @@
...
 
-In transactions, after Taxes and Charges table, you will find option to enter "Additional Discount Amount". Based on Amount entered in this field, item's Basic Rate and Taxes will be recalculated.
+<img alt="Discount Percentage" class="screenshot" src="{{docs_base_url}}/assets/image/articles/discount-2.png">
 
-Before applying Additional Discount Amount,
+If Discount Amount is applied based on the **Net Total**, then item's Net Rate and Net Amount is reduced as per the Discount Amount.
```
Comment (anandpdoshi) [inline erpnext/docs/user/manual/en/selling/articles/applying-discount.md]: This is too much detail. Just write *If ... then taxes are applied on the reduced net total*

File: erpnext/docs/user/manual/en/selling/articles/applying-discount.md [hunk 7]
```diff
@@ -1,31 +1,25 @@
...
 
-![Discount]({{docs_base_url}}/assets/img/articles/Selection_0085ca13e.png)
-
-After applying Additional Discount Amount.
-
-![Discount Amount]({{docs_base_url}}/assets/img/articles/Selection_010496ae2.png)
+If Discount Amount is applied based on the **Grand Total**, then with item's Net Rate, Net Amount and taxes are also re-calculated as per Discount Amount.
```
Comment (anandpdoshi) [inline erpnext/docs/user/manual/en/selling/articles/applying-discount.md]: *If ... then discount is applied on both Net Total and Taxes*

File: erpnext/docs/user/manual/en/selling/articles/erpnext-for-services-organization.md [hunk 8]
```diff
@@ -1,28 +1,45 @@
...
 
-https://conf.erpnext.com/2014/videos/umair-sayyed
+About 30% of ERPNext customers are companies into services. These are companies into software development, certification services, individual consultants and many more. Being into service business ourselves, we use ERPNext to manage our sales, accounting, support and HR operations. Check following video to learn how ERPNext uses ERPNext.
+
+<iframe width="640" height="360" src="//www.youtube.com/embed/b6r7WxJMfFA" frameborder="0" allowfullscreen=""></iframe>
 
 ###Master Setup
 
-Between the service and trading company, the most differentiating master is an item master. While trading and manufacturing business has stock item, with warehouse and other stock details, service items will have none of these details.
+Between the service and trading company, the differentiating master is an **Item** master. Service companies items are non-stock and service items. They don't have stock and warehouse to them.
```
Comment (anandpdoshi) [inline erpnext/docs/user/manual/en/selling/articles/erpnext-for-services-organization.md]: Avoid using the term `master`. 

Better:
*The setup for a Service company differs primarily for Items. They don't maintain the Stock for Items and thus, don't have Warehouses.*

File: erpnext/docs/user/manual/en/selling/articles/sales-persons-in-the-sales-transactions.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+<img class="screenshot" alt="Sales Person Customer" src="{{docs_base_url}}/assets/img/selling/sales-person-transaction-1.png">
+
+####Sales Person Contribution
+
+If more than one sales persons are working together on an order, then contribution % should also for each Sales Person, based on their effort.
+
+<img class="screenshot" alt="Sales Person Order" src="{{docs_base_url}}/assets/img/selling/sales-person-transaction-2.png">
+
+On saving transaction, based on the Net Total and %Contriution, Contribution to Net Total will be calculated for each Sales Person.
```
Comment (anandpdoshi) [inline erpnext/docs/user/manual/en/selling/articles/sales-persons-in-the-sales-transactions.md]: Typo

**Pull request merged**